test.rb

```diff
@@ -17,12 +17,12 @@ def interpolation
   end
 
   def condition
-    if true then 1 else 2 end
     true ? 1 : 2
+    if true then 1 else 2 end
   end
 
   def bang
-    !! false
+    !!!! false
```

Comment:
Avoid the use of double negation (`!!`).<br>Do not leave space between `!` and its argument.